components/grid/compact-grid.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Grid Size - Compact Grid
+page_title: Grid Size | Grid for Blazor
+description: Adjust the size and modify the appearance of the Telerik UI for Blazor Grid.
+slug: compact-grid
+tags: telerik,blazor,grid,size,compact
+published: true
+position: 20
+---
+
+## Grid Size
+
+This feature attempts to address the need for a **Compact Grid**, which renders more items by utilizing the available space, mainly through setting smaller padding in its cells.
+
+To utilize this feature, set the [`Size`](#grid-size) property of the component. It accepts values of type [`ThemeConstants.Grid.Size`](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.ThemeConstants.Grid.Size) and the supported values are: 
+
+1. `Small` - Sets the padding of the building blocks of the Grid to **2px 8px**.
+1. `Medium` (Default) - Sets the padding of the building blocks of the Grid to **4px 8px**.
+
+Changing the `Size` property affects different building blocks of the component (tables, buttons, inputs, dropdowns, and others). 
+
+> The `Size` option does not affect elements displayed inside a popup (such as [Filter Menu](https://docs.telerik.com/blazor-ui/components/grid/filter/filter-menu), [Column Menu](https://docs.telerik.com/blazor-ui/components/grid/columns/menu)). 
```

Comment:
Here we can explain why the size setting does not affect the popups - they are technically outside of the Grid but this may not be clear to all readers.

We should also list possible options to handle that in case they want to have small size for the items in teh popups - e.g. use templates.

I also suggest we list that in a "Notes" section together with the template specifics and the Grid with Toolbar example.